### DISCUSSION TITLE
Tax Residence Certificate

### DISCUSSION THREAD

**Discussion by mfrankiewicz** [community/community#137706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
can I have GitHub Tax Residence Certificate?

All the best!

**Comment by ebndev** [comment]
Hi @mfrankiewicz,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

If this is regarding GitHub Sponsors you might find this doc helpful: [Tax information for GitHub Sponsors](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/tax-information-for-github-sponsors)

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
